docs/extras/modules/data_connection/vectorstores/integrations/closevector.mdx

```diff
@@ -0,0 +1,51 @@
+import CodeBlock from "@theme/CodeBlock";
+
+# CloseVector
+
+:::tip Compatibility
+available on both browser and Node.js
+:::
+
+[CloseVector](https://closevector.getmegaportal.com/) is a cross-platform vector database that can be currently ran on both browser and Node.js. For example, you can create your index on Node.js and then load/query it on browser. For more information, please visit [CloseVector Docs](https://closevector-docs.getmegaportal.com/).
```

Comment:
s/that can be currently ran on both browser and Node.js/that can run in both the browser and Node.

Reply:
thanks, fixed that